src/stratigraphy/benchmark/ground_truth.py

```diff
@@ -29,9 +29,21 @@ def __init__(self, path: Path):
                 if parse_text(layer["material_description"]) != ""
             ]
             metadata = ground_truth_item["metadata"]
+
+            # Add the groundwater to the metadata if it is present
+            if "groundwater" in ground_truth_item:
+                metadata["groundwater"] = ground_truth_item["groundwater"]
```

Comment:
I'm not such a fan of including groundwater data with the "metadata". For me, the groundwater data belongs to the main geological content of the borehole profile, not to the metadata. Therefore, I would rather keep them separate in the ground truth as well.

Reply:
I totally agree. I refactored the code to change this in our presentation.